modules/reference/pages/properties/topic-properties.adoc

```diff
@@ -233,8 +236,54 @@ The minimum ratio between the number of bytes in dirty segments and the total nu
 
 ---
 
+[[max.compaction.lag.ms]]
+=== max.compaction.lag.ms
+
+The maximum amount of time (in ms) that a log segment can remain unaltered before it is eligible for compaction in a compact topic. Overrides the cluster property xref:cluster-properties.adoc#max_compaction_lag_ms[`max_compaction_lag_ms`] for the topic.
+
+*Unit:* milliseconds
+
+*Requires restart:* No
+
+*Visibility:* `user`
+
+*Type:* integer
```

Comment:
@Feediver1 we don't need to include Requires restart for topic properties. And I don't think we need Visibility either. None of the other topic properties list them.

Reply:
We probably don't need to include Unit either. It's in the property name, and most (not all) other topic properties do not include Unit.